### PR TITLE
corrected all of the mistakes(i hope) Mareike pointed out

### PR DESCRIPTION
**Pull request by nikstaaar**

### CODE REVIEW

File: html/index.html [hunk 1]
```diff
@@ -1,26 +1,27 @@
...
 <head>
     <title>My Personal Website</title>
-    <style></style>
 </head>
 <body>
     <header>
-        <img src="/media/cc0-images/grapefruit-slice-332-332.jpg">
+        <img alt="Grapefruit slice atop a pile of other slices" src="/media/cc0-images/grapefruit-slice-332-332.jpg">
         <nav class="quicknav">
             <a href="#News">News</a>
```
Comment (mbosselmann) [inline html/index.html]: You need to adjust `#News` as below you wrote `id="news"`. :-) So `#news` it is.

File: html/index.html [hunk 2]
```diff
@@ -1,26 +1,27 @@
...
             <a href="#contact">Contact</a>
             <a href="#about">About</a>
        </nav>
-        <div class = "searchbar">
-          <input type="text" placeholder="Search..">
-        </div>
+        <form>
+            <li class="button">
+                <button type="submit">Send your message</button>
+            </li>
```
Comment (mbosselmann) [inline html/index.html]: Somehow your input element is now missing. Please check again the article I recommended to you in my last code review about the html structure of forms. 🙏  As for the challenge itself, it is not necessary to add a form, you can also remove this part for now and add it later after our session about forms (planned for next week). :)

File: html/index.html [hunk 3]
```diff
@@ -1,26 +1,27 @@
...
         <nav class="quicknav">
             <a href="#News">News</a>
             <a href="#contact">Contact</a>
             <a href="#about">About</a>
        </nav>
-        <div class = "searchbar">
-          <input type="text" placeholder="Search..">
-        </div>
+        <form>
+            <li class="button">
```
Comment (mbosselmann) [inline html/index.html]: You won't use a `li` element without a `ul` or `ol` element. If you want to style the button you can add the class selector to the `button` element directly.

**Review by mbosselmann — COMMENTED** [review]
Nice job! 👍 Three comments I have left for you to look at.

And I have one more tip about the Git workflow. If you have created a pull request and you get a review, then you can add your changes to that pull request. For that you don't need to merge it, but make your changes locally in your feature branch, stage, commit and push those changes and they will be added to this pull request. Feel free to try this out in this pull request. :)

**Comment by nikstaaar** [comment]
Hi, I added all the changes to this pull requested as you suggested. I think we can merge for now? Thank you for all your feedback!

**Review by mbosselmann — APPROVED** [review]
I approve your PR. ✅  Nothing else to complain, just one thing: If code is not needed then it should be removed and not commented out. But this is ok for now as we haven't spoken about comments yet. Just keep this in mind for your next PR. :)

**Pull request merged**